DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -22,10 +22,19 @@ export class SearchSortFilterAndPaginate {
 
   private readonly filterEnabled: boolean;
 
+  private readonly searchEnabled: boolean;
+
+  private readonly sortEnabled: boolean;
+
+  private readonly paginationEnabled: boolean;
+
   // Route proved should be a relative path with no leading /
-  constructor(route: string, filterEnabled = false, filterCookieName = '') {
+  constructor(route: string, filterEnabled = false, filterCookieName = '', searchEnabled = true, sortEnabled = true, paginationEnabled = true) {

```

Comment:
I'm not sure about true being the default value for all of these (particularly search). I know it means you have to change less files in this MR, but I'm not sure if it the expected behaviour when we already have a filterEnabled defaulting to false.